src/v2/openapi.yaml

```diff
@@ -179,64 +178,56 @@ components:
         url:
           type: string
           example: https://snyk.io/vuln/SNYK-PYTHON-DJANGO-4002
-      description: Limited private vulnerability details for free tier users
-
-    PremiumVulnerabilityFields:
```

Comment:
Does it not make sense to keep this distinction between premium and basic fields. It will make it easier to distinguish it in the API server.

Reply:
It is not good to differentiate vulnerability fields at two place ie., backbone and API.
Considered use-case, say our agreement on this fields changes for free / registered user (one field is moved from registered user to free user), we need to modify Backbone and API server again to keep it in sync. 
So, I recommend to keep the field distinction at one place i.e., API server, so that we have to maintain only API server for distinction part.

Reply:
Ok